code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -341,6 +375,10 @@
 	using_special = FALSE
 	add_dragon_overlay()
 
+/mob/living/simple_animal/hostile/space_dragon/get_status_tab_items()
+	. = ..()
+	. += "Plasma: [plasma]/[MAXIMUM_STORED_PLASMA_MOLES]"
```

Comment:
it seems weird to store this in moles and not just shot counts

Reply:
Most of the time the dragon would be firing without its full effectiveness. It would seem weird to say "0.2 shots" or something.